scripts/pylib/twister/twisterlib/testsuite.py

```diff
@@ -41,13 +52,15 @@ def __init__(self,
                  has_registered_test_suites: bool = False,
                  has_run_registered_test_suites: bool = False,
                  has_test_main: bool = False,
-                 ztest_suite_names: List[str] = []):
+                 ztest_suite_names: List[str] = [],
+                 test_map: Dict[str, List[str]] = None):
```

Comment:
What is a purpose of storing those information? At this moment I can't see any meaningful example of using those data in code.